README.md

```diff
@@ -1,19 +1,62 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to pause Scalar DL Ledger cluster or Scalar DB server cluster to create a transactionally-consistent backup.
 
-## Use the client-side tool
+## scalar-admin client tool
+
+As of the current implementation, the scalar-admin client tool is mainly used to pause (and unpause) the cluster of applications that implement the admin interface.
+
+### Install
+
+#### Fat jar
+
+The release versions of `scalar-admin` fat jar can be downloaded from [releases](https://github.com/scalar-labs/scalar-admin/releases) page of Scalar Admin.
+
+#### Docker container
+
+You can pull the docker image from [Scalar's container registry](https://github.com/orgs/scalar-labs/packages/container/package/scalar-admin).
+
+#### Build from sources
+
+In case you want to build `scalar-admin` fat jar from the source:
+
+```console
+$ ./gradlew shadowJar
+```
+
+* The built fat jar file is `build/libs/scalar-admin-<version>-all.jar`
+
+In case you want to build the docker image from the source:
+
+```console
+$ ./gradlew docker
+```
+
+### Usage
+
+You can manage the scalar-admin integrated applications in the following ways
+
+#### Fat jar
+
+You can run the flat jar as follows.
```

Comment:
```suggestion
You can run the fat jar as follows.
```

Reply:
Please re-read the doc before asking for a review.

Reply:
Sorry for the inconvenience.
The above-specified suggestion is integrated.